modules/widgets/social-media-icons.php

```diff
@@ -116,6 +118,12 @@ public function widget( $args, $instance ) {
 			$html .= '<li><a title="' . sprintf( $alt_text, esc_attr( $instance['vimeo_username'] ), 'Vimeo' ) . '" href="' . esc_url( 'https://vimeo.com/' . $instance['vimeo_username'] . '/' ) . '" class="genericon genericon-vimeo" target="_blank"><span class="screen-reader-text">' . sprintf( $alt_text, esc_html( $instance['vimeo_username'] ), 'Vimeo' ) . '</span></a></li>';
 		}
 
+		if ( ! empty( $instance['google_username'] ) ) {
+			$title_text = (is_numeric( $instance['google_username'] ) ? $alt_text_numeric : $alt_text);
+			$plus = ( (!is_numeric( $instance['google_username']) && substr($instance['google_username'], 0,1 ) != "+" ) ? "+" : "" );
```

Comment:
Funny you should say that ... I didn't edit the github version but I did my own ... One sec.